MonoGame.Framework.Content.Pipeline/MonoGame.Content.Builder.targets

```diff
@@ -36,8 +36,8 @@
     <!-- We do this here so we are compatible with xbuild -->
     <ItemGroup>
       <ContentReferences Include="@(MonoGameContentReference)">
-        <ContentOutputDir>$(ProjectDir)%(RelativeDir)\bin\$(MonoGamePlatform)</ContentOutputDir>
-        <ContentIntermediateOutputDir>$(ProjectDir)%(RelativeDir)\obj\$(MonoGamePlatform)</ContentIntermediateOutputDir>
+        <ContentOutputDir>$(ProjectDir)%(RelativeDir)\bin\$(MonoGamePlatform)\%(Filename)</ContentOutputDir>
```

Comment:
It was important to set the output folder and intermeditate output folder passed to MGCB.exe as ones that are unique to the mgcb being built. As otherwise, when you have multiple mgcb's in the same directory, they all end up outputting to the same folders, and this was screwing up the results of the build.